wodles/aws/aws_s3.py

```diff
@@ -36,6 +42,13 @@
 except ImportError:
     print('ERROR: boto3 module is required.')
     sys.exit(4)
+
+try:
+    import pyarrow.parquet as pq
```

Comment:
Why do we do this and we don't do the same for botocore? Similar question to before.

Reply:
Also related to the management of the `boto3` dependency, `botocore` is required for it. 
```
boto3==1.17.85
  - botocore [required: >=1.20.85,<1.21.0, installed: 1.20.85]
    - jmespath [required: >=0.7.1,<1.0.0, installed: 0.9.5]
    - python-dateutil [required: >=2.1,<3.0.0, installed: 2.8.1]
      - six [required: >=1.5, installed: 1.14.0]
    - urllib3 [required: >=1.25.4,<1.27, installed: 1.26.5]
  - jmespath [required: >=0.7.1,<1.0.0, installed: 0.9.5]
  - s3transfer [required: >=0.4.0,<0.5.0, installed: 0.4.2]
    - botocore [required: >=1.12.36,<2.0a.0, installed: 1.20.85]
      - jmespath [required: >=0.7.1,<1.0.0, installed: 0.9.5]
      - python-dateutil [required: >=2.1,<3.0.0, installed: 2.8.1]
        - six [required: >=1.5, installed: 1.14.0]
      - urllib3 [required: >=1.25.4,<1.27, installed: 1.26.5]
```
We apply the same logic for the `boto3` import in the lines above the ones of the `pyarrow` import.